lib/kino/data_table.ex

```diff
@@ -78,40 +83,153 @@ defmodule Kino.DataTable do
 
   defp normalize_tabular(tabular), do: tabular
 
+  defp init_reader!(tabular) do
+    with :none <- Table.Reader.init(tabular) do
+      raise ArgumentError, "expected valid tabular data, but got: #{inspect(tabular)}"
+    end
+  end
```

Comment:
Should we expose `Table.init_reader!(tabular)`?

Reply:
I would say no, because we most often can raise better error messages by doing it ourselves.